composer.json

```diff
@@ -4,15 +4,17 @@
     "keywords": ["event-loop", "asynchronous"],
     "license": "MIT",
     "require": {
-        "php": ">=5.4.0"
+        "php": ">=5.4.0",
+        "evenement/evenement": "^3.0||^2.0"
```

Comment:
Well given the signal handler only has an `on` and an `off` even called once we don't really need it. I'll refactor and remove it.